src/index.html

```diff
@@ -6,7 +6,455 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items"> 
+            <a href="#" class="nav__link is-active" >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              samsung
+            </a> 
+          </li>
+
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              smartphones
+            </a> 
+          </li>
+            
+          <li class="nav__items"> 
+            <a 
+              class="nav__link" 
+              href="#" 
+              data-qa="nav-hover"
+              >
+              laptops & computers
+            </a>
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              gadgets
+            </a> 
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              tablets
+            </a>
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              photo
+            </a> 
+          </li>
+            
+          <li class="nav__items">
+            <a class="nav__link" href="#">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="products products--position">
+        <div class="card" data-qa="card">
+          <div class="card__picture">
+            <img 
+              class="card__images" 
+              src="images/imac.jpeg" 
+              alt="imac"
+            >
+          </div>
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__reviews">
+            <div class="stars">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
```

Comment:
redundant empty lines. Use only with _multiline_ siblings
```suggestion
            <div class="stars">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```